### PR TITLE
SPV_EXT_replicated_composites

### PR DESCRIPTION
**Pull request by kpet**
Specification: https://github.com/KhronosGroup/SPIRV-Registry/pull/253

### CODE REVIEW

**Review by Naghasan — APPROVED** [review]

File: include/spirv/unified1/spirv.core.grammar.json [hunk 1]
```diff
@@ -4854,6 +4854,42 @@
...
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",        "name" : "'Value'" }
+      ],
+      "capabilities" : [ "ReplicatedCompositesEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpCompositeConstructReplicateEXT",
+      "class"  : "Constant-Creation",
```
Comment (jeffbolznv) [inline include/spirv/unified1/spirv.core.grammar.json:4883]:
```suggestion
      "class"  : "Composite",
```

Comment (kpet) [reply include/spirv/unified1/spirv.core.grammar.json:4883]: Done for the two "Composite" instructions.

Comment (jeffbolznv) [reply include/spirv/unified1/spirv.core.grammar.json:4883]: This looks backward. The corresponding existing instructions are:
- OpConstantComposite and OpSpecConstantComposite are in "Constant-Creation Instructions"
- OpCompositeConstruct is in "Composite Instructions".

File: include/spirv/unified1/spirv.core.grammar.json [hunk 2]
```diff
@@ -4854,6 +4854,42 @@
...
```
Comment (jeffbolznv) [inline include/spirv/unified1/spirv.core.grammar.json:4883]:
```suggestion
      "class"  : "Composite",
```

File: include/spirv/unified1/spirv.core.grammar.json [hunk 3]
```diff
@@ -4854,6 +4854,42 @@
...
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",        "name" : "'Value'" }
+      ],
+      "capabilities" : [ "ReplicatedCompositesEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpSpecConstantCompositeReplicateEXT",
+      "class"  : "Composite",
```
Comment (jeffbolznv) [inline include/spirv/unified1/spirv.core.grammar.json:4871]:
```suggestion
      "class"  : "Constant-Creation",
```

File: include/spirv/unified1/spirv.core.grammar.json [hunk 4]
```diff
@@ -4854,6 +4854,42 @@
       "capabilities" : [ "CooperativeMatrixKHR" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpConstantCompositeReplicateEXT",
+      "class"  : "Composite",
```
Comment (jeffbolznv) [inline include/spirv/unified1/spirv.core.grammar.json:4859]:
```suggestion
      "class"  : "Constant-Creation",
```

**Review by jeffbolznv — APPROVED** [review]

**Comment by bashbaug** [comment]
Merging as discussed in the May 29th teleconference.

**Pull request merged**